src/game_initialization/create_engine.cpp

```diff
@@ -404,6 +404,12 @@ void create_engine::prepare_for_campaign(const std::string& difficulty)
 	state_.classification().campaign = current_level_data["id"].str();
 	state_.classification().campaign_name = current_level_data["name"].str();
 	state_.classification().abbrev = current_level_data["abbrev"].str();
+	if (current_level_data["type"] == "hybrid" && state_.classification().is_multiplayer()) {
+		// for hybrid campaigns in MP mode let's make a clarification in the abbrev
+		// so saves for sp and mp runs don't get confused
+		state_.classification().abbrev = state_.classification().abbrev + _(" MP");
```

Comment:
Based on existing convention, use - instead of space before MP. Might accidentally get flagged on some OSes.
```suggestion
		state_.classification().abbrev = state_.classification().abbrev + _("-MP");
```

Reply:
Flagged?

Reply:
I don't mind, I myself had doubt about the space. But there are spaces in the scenario names anyway, and we don't do anything about it?